assets/plugins/ytdl.js

```
@@ -83,29 +83,38 @@ command(
+}, async (message, match) => {
+    try {
+        match = match || message.reply_message?.text;
+        if (!match) return await message.reply("Give me a query");
+        const findUrl = `https://diegoson-naxordeve.hf.space/tubidy/search?q=${match}`;
+        const search = await fetch(findUrl);
+        const dlink = await search.json();
+        if (!dlink || !dlink.length) return await message.reply("Not_found");
+        const toBuffer = dlink[0];
+        if (!toBuffer.link) return;
+        await message.reply(`_Downloading ${toBuffer.title}_`);
+        const dlink = `https://diegoson-naxordeve.hf.space/tubidy/dl?url=${toBuffer.link}`;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'dlink'. Consider to delete it or rename it.

'dlink' is defined here:


(lint/suspicious/noRedeclare)